SwanCustomEnvironments/swancustomenvironments/scripts/makenv.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+
+# Author: Rodrigo Sobral 2024
+# Copyright CERN
+# This script allows to create an environment to use in notebooks and terminals. The environment contains the packages from a provided repository.
+
+LOG_FILE=/tmp/makenv.log
```

Comment:
Add a comment saying this file will contain the output / error of this script for possible debugging.